README.md

```diff
@@ -79,6 +79,10 @@ working with Memory, and see how the kernel works.
      [Polyglot extension](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.dotnet-interactive-vscode)
      to run the notebooks.
 
+### Working with Secrets
+
+We need keys to work with various aspects of the project including accessing openAI models. This opens up the possibility of exposing keys in commits. There are a [couple of options](https://learn.microsoft.com/en-us/aspnet/core/security/app-secrets?view=aspnetcore-7.0&tabs=windows) to safe guard developers from exposing keys. Outside of using the Secret Manager and environment variables, we've also added *.development.config to the .gitignore if developers want to use config files for secret storage.
```

Comment:
Apologies - I didn't notice where this file was earlier. Let's keep this level of detail to the READMEs of the apps themselves and not in the top-level README.